src/functions-and-arrays.js

```diff
@@ -1,19 +1,85 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+
+function maxOfTwoNumbers(num1, num2) {
+  if (num1 > num2) {
+    console.log(num1);
+    return num1;
+  }
+  else if (num1 < num2){
+    console.log(num2);
+    return num2;
+  }
+
+  else {
+    return num1;
+  }
+
+}
+
+maxOfTwoNumbers(45,56);
 
 
 
 // Iteration #2: Find longest word
+
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+
+
+function findLongestWord(words) {
+   let longestWord =''
+   
+     for (let i=0; i<words.length ; i++) 
+     {
+      if (longestWord.length < words[i].length) {
+      longestWord = words[i]
+      return longestWord;
+      }
+      
+    }
+    
+    if (words.length === 0) {
+      return null;
+    }
+    console.log (longestWord);
+    
+  }
+  
+  findLongestWord(words);
+  
+
 
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers() {
```

Comment:
On this function you should add a parameter, meaning that it should look like this:
```
function sumNumbers(arr) {
  let sum = 0;
  for (i=0; i < arr.length; i++ ) {
      sum += arr[i]
  }
  console.log (sum);
  return sum;
}
```
Besides that, the whole logic was perfect